keras_cv/losses/focal_test.py

```diff
@@ -68,13 +55,15 @@ def test_output_shape_from_logits(self):
         )
 
     def test_from_logits_argument(self):
-        y_true = tf.random.uniform((2, 8, 10))
-        y_logits = tf.random.uniform((2, 8, 10), minval=-1000, maxval=1000)
-        y_pred = tf.nn.sigmoid(y_logits)
+        np.random.seed(1337)
+        y_true = np.random.uniform(size=(2, 8, 10))
+        y_logits = np.random.uniform(size=(2, 8, 10), low=-1000, high=1000)
```

Comment:
Rather than setting the global seed (now legacy in NumPy), we can now do:

```suggestion
        rng = np.random.default_rng(1337)
        y_true = rng.uniform(size=(2, 8, 10))
        y_logits = rng.uniform(low=-1000, high=1000, size=(2, 8, 10))
```